### ISSUE TITLE
the setting of pretrained model on Synthtext

### ISSUE THREAD

**Issue by zhangchurong** [MhLiao/DB#109]
I am pretraining DB on Synthtext, the label is organized as eight points of each word. the remained setting is followed the setting of td500 as you recommended. But after 2 epoches  in Synthtext, the loss is still above 1. I have already check the reliability of the gt, it works correctly. I wonder is it because of my wrong setting in the training process, can you give my any advice? 
Thanks a lot!

**Comment by ganggang233** [comment]
> I am pretraining DB on Synthtext, the label is organized as eight points of each word. the remained setting is followed the setting of td500 as you recommended. But after 2 epoches in Synthtext, the loss is still above 1. I have already check the reliability of the gt, it works correctly. I wonder is it because of my wrong setting in the training process, can you give my any advice?
> Thanks a lot!

Ｈi, I'm new to the training network. Can you share your pre-training script?

**Comment by zhangchurong** [comment]
> > I am pretraining DB on Synthtext, the label is organized as eight points of each word. the remained setting is followed the setting of td500 as you recommended. But after 2 epoches in Synthtext, the loss is still above 1. I have already check the reliability of the gt, it works correctly. I wonder is it because of my wrong setting in the training process, can you give my any advice?
> > Thanks a lot!
> 
> Ｈi, I'm new to the training network. Can you share your pre-training script?

I did not write any new training code, just processed the gt like the way they did to the other datasets. It would be okay for me to share my code , but it dosen't work well right now.

**Comment by ganggang233** [comment]
> > > I am pretraining DB on Synthtext, the label is organized as eight points of each word. the remained setting is followed the setting of td500 as you recommended. But after 2 epoches in Synthtext, the loss is still above 1. I have already check the reliability of the gt, it works correctly. I wonder is it because of my wrong setting in the training process, can you give my any advice?
> > > Thanks a lot!
> > 
> > 
> > Ｈi, I'm new to the training network. Can you share your pre-training script?
> 
> I did not write any new training code, just processed the gt like the way they did to the other datasets. It would be okay for me to share my code , but it dosen't work well right now.

Tank You for your reply

**Comment by zhangchurong** [comment]
I have solved it

**Comment by wuzuowuyou** [comment]
> I have solved it

How do you solve the problem of non convergence

**Comment by yh673025667** [comment]
> I have solved it
您好，请问您训练synthtext的时候的setting能分享一下吗

**Comment by deepseek** [comment]
@zhangchurong
